### ISSUE TITLE
Nidos no existentes

### ISSUE THREAD

**Issue by VapoOMG** [duhow/ProfesorOak#257]

**Comment by duhow** [comment]
Habría que replantear el problema de otra forma. Si tenemos 20 Pokémon, sobran 220 (por decir algo), por lo tanto no es viable.

Lo que habría que hacer entonces es:
- El usuario tendrá una lista de Pokémon en los cuales está interesado.
- ¿Hay algún Pokémon de los que me interesan?
- Enviar por privado si cumple las condiciones.

**Comment by VapoOMG** [comment]
Sí, pero no hablamos de Pokemon de los que no se haya registrado nido, sino Pokemon sobre los cuales no vas a encontrar nido, porque no existe nido de estos. Por ejemplo no existe nido de Chansey, y esto significa que ni esta semana, ni la siguiente, ni la siguiente, ni la siguiente... Va a disponer de nido. En esta categoría están otros como Dratini, Mareep, Larvitar... Son Pokemon que no se encuentran en nidos. Si acaso en "spawns" o "biomas", pero no en nidos. Más que nada porque caaaaaaaaaada persona nueva que entra hace la misma pregunta y al final acaban llenando el chat de preguntas al profesor y lo veo innecesario, por eso si te devuelve un mensaje con una lista de Pokemon que no puedes encontrar en nido (porque no existe nido de estos, porque está programado así el juego) ya dejarás de preguntar sobre el resto. A mí me parece útil en ese sentido. La gente acaba preguntando siempre aunque exista la lista de nidos de Oak, y por ello lo propuse xD

No sé si me he explicado bien con lo de Pokemon sobre los que no existe nido pero comparándolo sería algo así:

Charmander: puede ser encontrado en nido.
Tauros: no puede ser encontrado en nido.
Sunkern: puede ser encontrado en nido.
Onix: puede ser encontrado en nido.
Unown: no puede ser encontrado en nido.
Ditto: no puede ser encontrado en nido.

Hay un número limitado de Pokemon que no pueden ser encontrados en nido, por eso creo que es viable, porque a lo mejor de 150 Pokemon (no cuento evoluciones porque los nidos de Pokemon evolucionados no existen, a excepción de aquellos evolucionados de un Pokemon bebé: Smoochum -> Jynx, Elekid -> Electabuzz), unos 15-20 son aquellos que no pueden ser encontrados en nido. Te puedo intentar hacer una lista así a ojo.

De primera gen: regionales (Tauros, Mr Mime, Kangaskhan y Farfetch'd), Koffing, Grimer, Porygon, Chansey, Dratini, Lickytung, Ditto.

De segunda gen: regionales (Corsola, Heracross), Mareep, Unown, Smeargle (todavía no está en el juego, pero si es como el caso de Ditto no tendrá nido), Miltank, Larvitar, Skarmory.

No sé si me dejo alguno pero creo que son todos esos los que no puedes encontrar en nido, 19 contados. Los legendarios y bebes no los incluiría en esta lista porque los legendarios no creo que estén sueltos como si de Pidgey se tratase y los bebés por alguna razón nadie pregunta por nido de ellos así que deben de haber comprendido que no existe nido sobre los Pokemon bebés (ni siquiera se encuentran salvajes).

No sé, lo veo una función útil para, sobretodo, evitar una cadena de preguntas innecesarias del estilo "Donde hay Chansey?" o "Donde hay Unown?" y un largo etc...

Sería más bien un mensaje plano y no uno en el que el bot busque en las listas de nidos, para el que las palabras clave sea uno de la lista de Pokemon que no puedes encontrar en un nido, sino se volvería loco XD

Espero no ser insistente con ello pero creo que de verdad es útil y hasta cierto punto "necesario" :)
